src/KMonad/Keyboard/Keycode.hs

```diff
@@ -305,6 +305,9 @@ data Keycode
   | KeyFn
   | KeyLaunchpad
   | KeyMissionCtrl
+#endif
+#ifdef mingw32_HOST_OS
+  | KeyPrintLegacy
```

Comment:
> What I am not sure about is whether to keep lagacy print key. Does anyone use that key? Should we remove it and add it only when a user asks for it?

I'd say the less `CPP` we use the better, but it's already enabled for this file anyways so...  I don't really have an opinion, as I'm not a Windows user.  I'll go with whatever people who use it say would be the best option